core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/handler/ExportToOtherGraphHandler.java

```diff
@@ -46,97 +47,146 @@ protected OtherGraphExporter createExporter(final ExportToOtherGraph export, fin
     }
 
     protected Graph createGraph(final ExportToOtherGraph<?> export, final Store store) {
+        validationHandling(store, export);
 
         final String exportGraphId = export.getGraphId();
```

Comment:
This was originally like that, as described above, but to limit the usage of calls we decided to pass individual variables.  I don't disagree with this though.